charts/zookeeper-operator/templates/pre-delete-hooks.yaml

```diff
@@ -3,6 +3,7 @@ kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: {{ template "zookeeper-operator.fullname" . }}-pre-delete
+  namespace: {{ .Release.Namespace }}
```

Comment:
namespace is not required for clusterrole/clusterrole binding, as it is not attached to namespace

Reply:
I see, thanks @anishakj! So we should also remove the namespace field from the clusterrole and clusterrolebinding within the `post-install-upgrade-hooks.yaml` as well?

Reply:
@panyuenlau  Not seeing `clusterRole` in `post-install-upgrade-hooks.yaml`

Reply:
@anishakj  oops my bad, I was looking into a wrong file